app/controllers/contacts_controller.rb

```diff
@@ -0,0 +1,26 @@
+class ContactsController < ApplicationController
+
+  def new
+    @contact = Contact.new
+  end
+
+  def create
+    @contact = Contact.new(contact_params)
+    @contact.company_id = params[:company_id]
+    @company = @contact.company
+    if @contact.save
+      flash[:success] = "You created a new contact!"
+      redirect_to company_jobs_path(@company)
+    else
+      render :new
```

Comment:
same here - if a user enters invalid data, we want to show them a nice error message - don't forget to handle your errors!